### ISSUE TITLE
Should be able to execute tests without exception from taiko

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#332]
**Expected behavior**
Should be able to execute tests without exception from taiko

**Actual behavior**
```
Error Message: Error: Failed to launch chrome!
    [115139:115139:0109/201735.568829:ERROR:process_singleton_posix.cc(280)] Failed to create /tmp/taiko_dev_profile-htSCqw/SingletonLock: No such file or directory (2)
    [115139:115139:0109/201735.568884:ERROR:chrome_browser_main.cc(1413)] Failed to create a ProcessSingleton for your profile directory. This means that running multiple instances would start multiple browser processes rather than opening a new window in the existing process. Aborting now to avoid profile corruption.
    
    Stacktrace: 
    Error: Failed to launch chrome!
    [115139:115139:0109/201735.568829:ERROR:process_singleton_posix.cc(280)] Failed to create /tmp/taiko_dev_profile-htSCqw/SingletonLock: No such file or directory (2)
    [115139:115139:0109/201735.568884:ERROR:chrome_browser_main.cc(1413)] Failed to create a ProcessSingleton for your profile directory. This means that running multiple instances would start multiple browser processes rather than opening a new window in the existing process. Aborting now to avoid profile corruption.
    
        at onClose (node_modules/taiko/lib/browserFetcher.js:222:24)
        at Interface.helper.addEventListener (node_modules/taiko/lib/browserFetcher.js:210:60)
        at emitNone (events.js:111:20)
        at Interface.emit (events.js:208:7)
        at Interface.close (readline.js:370:8)
        at Socket.onend (readline.js:149:10)
        at emitNone (events.js:111:20)
        at Socket.emit (events.js:208:7)
        at endReadableNT (_stream_readable.js:1064:12)
```
**Steps to replicate**
> Observed on Linux 4.15.0-29-generic Ubuntu 18.04.1 LTS, 2 process cores, 4096 MB with 20GB Hard disk

* Project - https://github.com/getgauge/taiko/tree/master/test/functional-tests
* Run `gauge specs --tags=\!knownIssue`
> Almost all tests fail with the above error

**Version**
```
commit - ef46f512b661ac149e1a9af482257d4b0d4abcef
```

**Comment by sguptatw** [comment]
The issue seems to be that chromium wasn't started successfully. One of the users had posted this https://n00bsys0p.wordpress.com/2011/05/26/google-chrome-failed-to-create-a-processsingleton-for-your-profile-directory-fix/ for chrome. Hope this helps to investigate further.

**Comment by nehashri** [comment]
The issue pointed seems to be because of files in tmp folder because of which Chome cannot start. This is something that the users will have to handle on their part. 

I tried to reproduce this with lower spec Ubuntu box. I don't seem to be able to reproduce this however. Another point to note is that we have upgraded our chromium versions, so with the latest commit (c487ea4ef01f72e4e6b759d2328b61c9cc04b479) it doesn't seem to be reproducible.
